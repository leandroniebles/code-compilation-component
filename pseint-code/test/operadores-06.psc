Proceso sin_titulo
	Escribir 3+(-2);
	Escribir 3^(-2);
	Escribir +3==-2;
	Escribir (+3)==(-2);
FinProceso
