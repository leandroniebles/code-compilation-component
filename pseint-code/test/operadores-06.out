1
0.1111111111
FALSO
FALSO
